packages/core/admin/admin/src/tests/StrapiApp.test.tsx

```diff
@@ -44,13 +44,16 @@ describe('ADMIN | new StrapiApp', () => {
           },
         },
         "content-manager_listView": {
-          "components": [],
-          "contentType": {},
+          "components": {},
+          "contentType": null,
           "data": [],
           "displayedHeaders": [],
           "initialDisplayedHeaders": [],
           "isLoading": true,
           "pagination": {
+            "page": 0,
```

Comment:
Shouldn't be page 1 by default ? (I don't think there's that much impact either way)

Reply:
I thought that, but it felt more appropriate to have all the pagination defaults as 0:

```ts
 pagination: {
    page: 0,
    pageCount: 0,
    pageSize: 0,
    total: 0,
  },
```

Otherwise if page was `1` i guess pageCount should be `1` too? then all of them should be 1 🤔 at least 0 signifies "this isn't right" – what do you think?